docs/plugins/devtools.md

```diff
@@ -39,7 +39,10 @@ The plugin supports the following options passed via the `forRoot` method:
 - `name`: Set the name by which this store instance is referenced in devtools (Default: 'NGXS')
 - `disabled`: Disable the devtools during production
 - `maxAge`: Max number of entries to keep.
-- `actionSanitizer`: Reformat actions before sending to dev tools
```

Comment:
The `actionSanitizer` line should not have been removed.
Unless there is a reason for removing this?

Reply:
@markwhitfeld added back